vault/resource_quota_lease_count_test.go

```diff
@@ -0,0 +1,90 @@
+package vault
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func randomQuotaLeaseString() string {
+	whole := acctest.RandIntRange(1000, 2000)
+	return strconv.Itoa(whole)
```

Comment:
For some reason RandIntRange is only returning a value within 0-1000 here, so adding the min should get it into 1000-2000.
```suggestion
	return strconv.Itoa(whole + 1000)
```

(Hopefully this is something addressed in a newer version of the SDK.)

Reply:
Updated